kasa/smart/modules/autooff.py

```diff
@@ -18,11 +17,17 @@ class AutoOff(SmartModule):
     REQUIRED_COMPONENT = "auto_off"
     QUERY_GETTER_NAME = "get_auto_off_config"
 
-    def __init__(self, device: SmartDevice, module: str):
-        super().__init__(device, module)
+    def _initialize_features(self):
+        """Initialize features after the initial update."""
+        if not isinstance(self.data, dict):
```

Comment:
What is `self.data` when the parent doesn't have the module data? I would have thought it would error with the check for update() logic in the `data` property but I guess not.

Reply:
The data is in this case the error itself.

Reply:
Ah, the SmartErrorCode exception I guess. We could probably improve how we handle those in a future PR.